### PR TITLE
SVN Trunk update and missing entrypoints

### LINKED ISSUE #3: Patch VICE to use XDG directories
Currently, VICE does not respect XDG paths and defaults to write to $HOME/.vice directory. This is less than ideal in the sandbox world. The code should be patched to use XDG instead.

### PR DESCRIPTION
**Pull request by digitalstreamio**
- pull in TingPing changes to build from SVN trunk
- update SVN trunk to latest revision
- add missing entrypoints
- fixes #3

### CODE REVIEW

File: net.sf.VICE.yml [hunk 1]
```diff
@@ -70,45 +69,72 @@ modules:
...
+        commands: [ sed 's/lib64/lib/g' -i vice/configure.proto ]
+
+  - name: appdata
+    buildsystem: simple
+    build-commands: 
       - install -Dm644 net.sf.VICE.appdata.xml /app/share/appdata/net.sf.VICE.appdata.xml
-      - install -Dm644 net.sf.VICE.desktop /app/share/applications/net.sf.VICE.desktop
       - install -Dm644 net.sf.VICE.png /app/share/icons/hicolor/128x128/apps/net.sf.VICE.png
       - install -Dm644 net.sf.VICE-64.png /app/share/icons/hicolor/64x64/apps/net.sf.VICE.png
+      - install -Dm644 net.sf.VICE.C64.desktop /app/share/applications/net.sf.VICE.C64.desktop
```
Comment (TingPing) [inline net.sf.VICE.yml]: `install -Dm644 -t /app/share/applications net.sf.VICE.*.desktop`

**Comment by TingPing** [comment]
Just to confirm upstream fixed the XDG issue?

File: net.sf.VICE.yml (hunk 1 above)
Comment (digitalstreamio) [reply net.sf.VICE.yml]: done

**Comment by digitalstreamio** [comment]
@TingPing - config/data files are now saved to ~/.var/app/net.sf.VICE so it looks like we are in good shape.

**Comment by digitalstreamio** [comment]
Yes, thank you for making these changes. Configuration files are now placed in their proper location. Temp files should probably go in XDG_CACHE_HOME instead. Cheers!

**Comment by digitalstreamio** [comment]
@TingPing I just noticed flathub is still pointing to the older version from Oct. Any idea why the build we pushed was never published?

**Comment by TingPing** [comment]
Failed to build: https://flathub.org/builds/#/builders/2/builds/6745

Triggering a new build: https://flathub.org/builds/#/builders/2/builds/6985

**Comment by TingPing** [comment]
Looks like a build server problem: https://github.com/flathub/flathub/issues/790

**Comment by digitalstreamio** [comment]
That makes sense. I saw the build kickoff and didn't catch that it didn't finish hence the confusion on my part.

**Pull request merged**